sycl/doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc

```diff
@@ -748,6 +762,101 @@ myQueue.submit([&](handler &cgh) {
 });
 ----
 
+== Host Side pipe read/write
+
+The read/write member functions of a host pipe have different signatures when they are called from the host side, in which case a `sycl::queue` is added to the parameters. `memory_order` is also added to the parameter for future work.
```

Comment:
I think we should not say that the `read` and `write` functions have different signatures on the host and target.  This won't be possible when we move to a 1-pass compiler (where both the host and device code are compiled with a single pass of the compiler).  However, we could have multiple overloads of the `read` and `write` functions, where some overloads are used from host code and some are used from device code.

Reply:
@gmlueck , thanks for the review. I have made the change accordingly.